### ISSUE TITLE
Bootstrap 3

### ISSUE THREAD

**Issue by martinpengellyphillips** [atoha/dbootstrap#69]
Bootstrap 3 RC1 is out (http://twitter.github.io/bootstrap/). Do folks want dbootstrap to update to this new theme? Or offer as an alternative (perhaps a different base class name)?

**Comment by jaetea** [comment]
Update to the new theme. Most people customize bootstrap so having two default themes is a waste.

**Comment by martinpengellyphillips** [comment]
Makes sense. I'll try to do #12 first and hopefully at the end of that it will be much easier to change to the newer style (and also for others to customise further).

**Comment by jaetea** [comment]
Any updates on this? BS3 has been finalized.

**Comment by martinpengellyphillips** [comment]
Hey. At present I probably won't have time to look at this for a couple of months.

**Comment by gavinr** [comment]
Any update on this? I'd love to have access to the latest build of FontAwesome.

**Comment by martinpengellyphillips** [comment]
No work done for this yet. Would like to wait until some of the core is structured better to actually allow building different versions of the themes using just changes to the variables. However, I am struggling to find time to work on this project at present so can't give any reliable timescales.

Pull requests welcome though.

**Comment by slimmtl** [comment]
Any one try to do this and succeed?
